modules/wifi/wifi_assoc.go

```diff
@@ -111,6 +111,9 @@ func (mod *WiFiModule) startAssoc(to net.HardwareAddr) error {
 		// send the association request frames
 		for _, ap := range toAssoc {
 			if mod.Running() {
+				if ap == nil {
+					continue
+				}
```

Comment:
this is a sub-optimal fix and its really a band-aid fix. Even if this check passes, it is entirely possible for `ap` to be nil a few lines down still causing the panic this is attempting to fix. Really this "fix" just makes it less likely for the problem to happen.